src/components/pages/MissionControl/RenderMissionControl.js

```diff
@@ -85,12 +85,13 @@ const RenderMissionControl = props => {
           setModalVisible(false);
         }}
         instructions={instructionText}
-        style={{ fontSize: '2rem' }}
+        className="mission-control-modal"
+        style={{ fontSize: '1.8rem' }}
         showOkButton={showButton}
       />
       <div className="mission-container">
         <Row className="main-row">
-          <Col className="read" xs={24} sm={12} onClick={handleReadStory}>
+          <Col className="read" span={24} onClick={handleReadStory}>
```

Comment:
In order to stack the components I had to change the flex method on the page - this is done in ant-design by adjusting the span of the columns.

Reply:
👍🏻